.nvmrc

```diff
@@ -1 +1 @@
-20
+24
```

Comment:
i think we also need to adjust something in the github workflows:

- https://github.com/vivid-planet/dev-process-manager/blob/main/.github/workflows/release.yml#L21
- https://github.com/vivid-planet/dev-process-manager/blob/main/.github/workflows/release-canary.yml#L22
- https://github.com/vivid-planet/dev-process-manager/blob/main/.github/workflows/lint.yml#L27

and maybe the engines in the package json:
- https://github.com/vivid-planet/dev-process-manager/blob/main/package.json#L62